modules/customize-certificates-add-service-serving-crd.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// * authentication/certificates/service-serving-certificate.adoc
+
+[id="add-service-certificate-crd_{context}"]
+= Add the service CA bundle to a Custom Resource Definition
+
+You can annotate a Custom Resource Definition (CRD) with `service.beta.openshift.io/inject-cabundle=true` to have its `caBundle` field populated with the service CA bundle. This allows the Kubernetes API server to validate the service CA certificate used to secure the targeted endpoint.
+
+[NOTE]
+====
+The service CA bundle will only be injected into the CRD if the CRD is configured to use a webhook for conversion that is secured with a service CA certificate.
```

Comment:
@marun Can you see if I got this wording accurate?

Reply:
The service CA bundle will only be injected into the CRD if the CRD is configured to use a webhook for conversion. It is only useful to inject the service CA bundle if a crd's webhook is secured with a service CA certificate.